### DISCUSSION TITLE
use non persistent home user directory

### DISCUSSION THREAD

**Discussion by JustinBridoux** [ltsp/ltsp#853] [Q&A]
Hi,

For 10 stations with same configuration, I create only one user and all station log in with that user.  But it's not stable, conflict bewteen them, so I'm looking for a way to use /home/user non permanent.  I think the best way is to include /home/user directory into ltsp image ? It's a good think to me that any changes lost on next reboot.

I find this post but didn't help me :
https://github.com/ltsp/ltsp/discussions/795

Any idea how to do that ?
Cheers,
Justin

**Comment by alkisg** [comment]
If you [search for guest](https://github.com/ltsp/ltsp/discussions?discussions_q=guest) you'll find many related discussions and solutions.
Each one has its benefits and drawbacks. In some cases the following benefits are desirable:

- To have a single guest account for any number of clients
- To be able to customize the /home/guest directory on the server
- The clients to be unable to see each other's files (for both security and stability reasons)
- The client home may:
  - Reside on the client, wasting RAM, but saving storage from the server; that's good for high-RAM clients and non-persistent homes
  - Reside on the server, wasting some storage but saving RAM from the client; that's good for low-RAM clients and/or persistent homes

Unfortunately I have no use case to implement all that, so either sponsoring or a PR will be needed for that solution to make it into upstream LTSP.
